Dockerfile

```diff
@@ -85,6 +87,10 @@ ENV NODE_RED_VERSION=$NODE_RED_VERSION \
     NODE_PATH=/usr/src/node-red/node_modules:/data/node_modules \
     FLOWS=flows.json
 
+#install custom nodes
+COPY data/nodes /data/nodes
+RUN ls /data/nodes | xargs -i npm install /data/nodes/{}
```

Comment:
I think we should use [local paths](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#local-paths) in `package.json` instead, like:

```
  "dependencies": {
    "node-red": "1.2.1",
    "lower-case": "file:./data/nodes/lower-case",
    "seed": "file:./data/nodes/seed",
  },
```

I think this may be the state of the `package.json` once your original script run inside the container.

Reply:
This is the tree it generates when finished yes ...

```
  "dependencies": {
    "lower-case": "file:../../../data/nodes/lower-case",
    "node-red": "1.2.1",
    "seed": "file:../../../data/nodes/seed"
  },
```